README.md

```diff
@@ -1,39 +1,51 @@
-# dblapi.js
-An official module for interacting with the top.gg API
+# TopGG.js
 
-## Installation
-`npm install dblapi.js`
+An official module for interacting with the Top.<span>GG API
 
-## Documentation
-Documentation can be found [here](https://top.gg/api/docs#jslib)
+# Installation
 
-## Example
+`npm i @top-gg/sdk`
+
+# Introduction
+
+The base client is TopGG.API, and it takes your top.<span>gg token and an optional Discord client.
+
+Your Top.<span>GG token can be found [here](https://top.gg/api/docs#mybots)
+
+If the client is provided, the poster will auto-retrieve statistics and post them to top<span>.gg, otherwise you can use the `API#postStats(serverCount, shardId, shardCount)` to do it manually.
+
+The client also comes with a suite of API interaction methods.
+
+# Popular examples
+
+## Auto-Posting stats
 
-### Example of posting server count with supported libraries (Discord.js and Eris)
 ```js
-const Discord = require("discord.js");
-const client = new Discord.Client();
-const DBL = require("dblapi.js");
-const dbl = new DBL('Your top.gg token', client);
-
-// Optional events
-dbl.on('posted', () => {
-  console.log('Server count posted!');
-})
+const client = Discord.Client() // Your discord.js or Eris client
 
-dbl.on('error', e => {
- console.log(`Oops! ${e}`);
+const TopGG = require('@top-gg/sdk')
+
+const api = new TopGG.API('Your top.gg token', client)
```

Comment:
Would recommend naming it Api as the class name instead to keep consistent with the lowerCamelCase naming scheme. The reason why this is the case is for it to properly convert to other cases (e.g. snake_case) it will be easier.

https://medium.com/fantageek/using-camelcase-for-abbreviations-232eb67d872